sdk/openai/azure-ai-openai-assistants/README.md

```diff
@@ -200,5 +343,5 @@ For details on contributing to this repository, see the [contributing guide](htt
 [azure_openai_access]: https://learn.microsoft.com/azure/cognitive-services/openai/overview#how-do-i-get-access-to-azure-openai
 [azure_subscription]: https://azure.microsoft.com/free/
 [azure_identity]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/identity/azure-identity
-
+[function_tool_call_full_sample]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/openai/azure-ai-openai-assistants/src/samples/java/com/azure/ai/openai/assistants/FunctionToolCallSample.java
```

Comment:
This link will exist when we merge to `main` I hope. We need to double check that after the actual merge!